pkg/redis/redis.go

```diff
@@ -145,7 +145,9 @@ func (r *Redis) ReadKey(ctx context.Context) error {
 
 			err := r.Pool.Do(radix.Cmd(&value, "DUMP", key))
 			if err != nil {
-				return err
+				fmt.Println(err)
+				fmt.Println(key)
+				continue
```

Comment:
if you only want to show error then please show error and stop, skip error is not a good way to do